test/correctness/require.cpp

```diff
@@ -64,6 +65,28 @@ static void test(int vector_width) {
 int main(int argc, char **argv) {
     test(0);
     test(4);
+
+    {
+        // Verify that the HVX backend can compile vectorized require() correctly
+        Target target("hexagon-32-noos-hvx_64");
+
+        Var x;
+        Func f;
+        f(x) = require(x > 0, x);
+        f.vectorize(x, 8).hexagon();
```

Comment:
Not that it should matter for this testcase nor am I suggesting that 8 is unsupported but idiomatically on Hexagon, one would want ```f.vectorize(x, 32).hexagon()```

Reply:
Done